templates/interface/add-markers.interface.hbs

```diff
@@ -37,7 +37,7 @@
           <template v-else>
             You are using the data source called <strong>\{{markersDataSource.name}}</strong>.<br>
             <span>
-              <a href="#" @click.prevent="editDataSource">Bulk upload data</a> | 
+              <a href="#" @click.prevent="editDataSource">Bulk upload data</a> |
```

Comment:
you sure this change hasn't removed a required space? If so, use `&nbsp;`

Reply:
Everything works as before.